powermeter/vzlogger.py

```
@@ -6,12 +6,18 @@ class VZLogger(Powermeter):
     def get_json(self):
-        url = f"http://{self.ip}:{self.port}/{self.uuid}"
+        url = f"http://{self.ip}:{self.port}"
         return self.session.get(url, timeout=10).json()
 
     def get_powermeter_watts(self):
-        return [int(self.get_json()["data"][0]["tuples"][0][1])]
+        powers = [0, 0, 0]
+        for item in self.get_json()["data"]:
+            uuid = item['uuid']
+            pos = self.uuids.get(uuid, None)
+            if pos != None:
```

Ruff (0.15.10)

[error] Comparison to `None` should be `cond is not None`

Replace with `cond is not None`

(E711)